### PR TITLE
Separate VCR & pytest concerns

### PR DESCRIPTION
**Pull request by tzturner**

### CODE REVIEW

**Review by edthedev — APPROVED** [review]

File: src/vcr_pytest.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import os
```
Comment (ddriddle) [inline src/vcr_pytest.py:1]: Do we need this code at all here?

**Review by ddriddle — APPROVED** [review]

**Review by mpitcel — DISMISSED** [review]

File: src/vcr_pytest.py (hunk 1 above)
Comment (mpitcel) [reply src/vcr_pytest.py:1]: I was wondering this as well

Comment (edthedev) [reply src/vcr_pytest.py:1]: Team discussion - let's get rid of this, but capture key lines of code like VCR().register_serializer( our_cleaner_serializer )

File: tests/test_vcr_cleaner.py [hunk 2]
```diff
@@ -1,31 +1,10 @@
...
-# TODO: Add unit tests. 
 
-# TODO: Think about how to get rid of need 
-# to export VCR_RECORD and deleting the YAML
-def test_with_vcr(cassette):
-
-    # Assemble
-    @cleaner(uri=f'{CASSETTE_ENDPOINT}/robots.txt')
+def test_with_vcr():
+    @cleaner(uri=f'helloworld.com/robots.txt')
```
Comment (edthedev) [inline tests/test_vcr_cleaner.py:7]:
```suggestion
 @clean_if(uri=f'helloworld.com/robots.txt')
```

**Review by edthedev — APPROVED** [review]

**Review by mpitcel — APPROVED** [review]

**Review by ddriddle — APPROVED** [review]

**Pull request merged**